docs/install_guides/pypi-openvino-dev.md

```diff
@@ -92,13 +99,13 @@ pip install openvino-dev[extras]
 | tensorflow                      |   [TensorFlow* 1.x](https://www.tensorflow.org/versions#tensorflow_1)            |
 | tensorflow2                     |   [TensorFlow* 2.x](https://www.tensorflow.org/versions#tensorflow_2)            |
 
-For example, to install and configure the components for working with TensorFlow 2.x, Apache MXNet and Caffe, use the following command:  
+For example, to install and configure the components for working with TensorFlow 2.x, Apache MXNet and Caffe, use the following command:
    ```sh
    pip install openvino-dev[tensorflow2,mxnet,caffe]
```

Comment:
Why is this different from the other page? Few are using caffe and mxnet today

Reply:
Agree. Replaced with TF2 and ONNX example.